library/Fission/Web/Domain.hs

```diff
@@ -0,0 +1,31 @@
+module Fission.Web.Domain
```

Comment:
Maybe `Fission.Web.DNS`? 🤔

Reply:
Yeah I had that thought yesterday as well. This confirms it so I'll change it out :ok_hand: